spec/fixtures/vcr/services/google_search/search_keyword.yml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
I think we can remove it as we are doing any HTTP request in this PR

Reply:
Fixed in [`4c447f6` (#42)](https://github.com/suho/google-search-ruby/pull/42/commits/4c447f699243eec5c046c00dd38c76964fc91c8f)